Cargo.toml

```diff
@@ -16,3 +16,7 @@ members = [
   "tower-service-util",
   "tower-timeout",
 ]
+
+[patch.'https://github.com/tower-rs/tower']
+tower-service = { path = "tower-service" }
```

Comment:
Why is this needed?

Reply:
ah this was a relic from tower-hyper depending on the git version but its fixed now. Removing it.